modules/concepts/hooks/list-of-hooks/displayProductPageDrawer.md

```diff
@@ -0,0 +1,27 @@
+---
+Title: displayProductPageDrawer
```

Comment:
again, it seems there is an issue with getting a source of the display hooks

Reply:
SAme it's only in the hook.xml file